korangar/src/main.rs

```diff
@@ -743,7 +747,12 @@ fn main() {
                             entities.push(npc);
                         }
                         NetworkEvent::RemoveEntity(entity_id) => {
-                            entities.retain(|entity| entity.get_entity_id() != entity_id);
+                            if entities[0].get_entity_id() == entity_id {
+                                entities[0].set_death(client_tick);
```

Comment:
Same here, `set_dead`, instead of `set_death`

Reply:
i will change